src/query/api/v1/handler/json/write.go

```diff
@@ -77,12 +77,18 @@ func (h *WriteJSONHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	writeQuery, err := newStorageWriteQuery(req)
 	if err != nil {
-		logging.WithContext(r.Context()).Error("Parsing error", zap.Any("err", err))
+		logger := logging.WithContext(r.Context())
+		logger.Error("parsing error",
+			zap.String("remoteAddr", r.RemoteAddr),
+			zap.Any("err", err))
 		xhttp.Error(w, err, http.StatusInternalServerError)
 	}
 
 	if err := h.store.Write(r.Context(), writeQuery); err != nil {
-		logging.WithContext(r.Context()).Error("Write error", zap.Any("err", err))
+		logger := logging.WithContext(r.Context())
+		logger.Error("write error",
+			zap.String("remoteAddr", r.RemoteAddr),
+			zap.Any("err", err))
```

Comment:
Nit: mind changing this to `Zap.Error(err)` in all the places

Reply:
Good call.